src/lv_drv_conf.h

```diff
@@ -85,18 +85,20 @@
  *-------------------*/
 
 /* SDL based drivers for display, mouse, mousewheel and keyboard*/
+#if defined(VOORKANT_LVGL_SDL)
```

Comment:
is it possible to put this somewhere else so that the lv_drv_conf.h stays the same as the out of the box, or at least the same lines? I think this might help us for maintenance later on.

Reply:
I just realised another option. just below the highlighted line, the file says (and already said):

```
#ifndef USE_SDL
# define USE_SDL 1
#endif
```

so we could remove our ifdefs, put a 0 there, and pass `-DUSE_SDL=1` from Meson. We can't override SDL *parameters* that way, but right now this allows us to clean up the file like you are suggesting. Thoughts?

Reply:
So, i'm sorry that you took so much time to look at htis. I"m find with lv_drv_conf.h is not 100% standard anymore. I would have liked it, but i don't think it's critical at all.

Reply:
It was a fine exercise :) so we'll leave it as it is in this PR now?